indico/htdocs/js/indico/modules/sessions.js

```diff
@@ -0,0 +1,149 @@
+(function(global) {
+    'use strict';
+
+    function handleRowSelection() {
+        $('.sessions-wrapper').on('change', '.select-row', function() {
+            $(this).closest('tr').toggleClass('selected', this.checked);
+            $('.js-requires-selected-row').toggleClass('disabled', !$('.sessions input:checkbox:checked').length);
+        }).trigger('change');
+    }
+
+    function setupTableSorter() {
+        $('.sessions .tablesorter').tablesorter({
+            cssAsc: 'header-sort-asc',
+            cssDesc: 'header-sort-desc',
+            cssInfoBlock: 'avoid-sort',
+            cssChildRow: 'session-blocks-row',
+            headerTemplate: '',
+            sortList: [[1, 0]]
+        });
+    }
+
+    function setupColorSwitches() {
+        $('.js-color-switch').each(function() {
+            var $this = $(this);
+            $this.colorswitch({
+                defaultColors: $this.data('colors'),
+                onSelect: function(background, text) {
+                    $.ajax({
+                        url: $(this).data('href'),
+                        method: $(this).data('method'),
+                        data: JSON.stringify({'colors': {'text': text, 'background': background}}),
+                        dataType: 'json',
+                        contentType: 'application/json',
+                        traditional: true,
```

Comment:
We don't need to specify `traditional: true` anymore since: https://github.com/indico/indico/commit/8f4572a22f68b7b16c155707b065e1a915770c63. It's still in master only. Can you cherry-pick it?